### ISSUE TITLE
✨Feature request:  Save&&close

### ISSUE THREAD

**Issue by dnknn** [allen505/save-for-later-firefox#14]
<img align=right width=30% src=https://user-images.githubusercontent.com/10995302/108611534-5ccf6700-741a-11eb-9b4e-fbd777d6a712.png>

The specific functions are shown in →_→ img:
But the way to achieve it is up to you, for example:

　
　
- or not add a button, but the following way will be better 👇!

![212_21080406](https://user-images.githubusercontent.com/10995302/108611638-60afb900-741b-11eb-8c21-604db2a6aa38.png)

**Comment by allen505** [comment]
Hi @dnknn . This is a nice feature request. So what we can do here is to add it as Settings option and check if we need to close the window/tabs saved. Also if the entire window is saved, then we would also need to have to open a new tab to keep the browser running.   
Thanks for the feature idea!

**Comment by dnknn** [comment]
> So what we can do here is to add it as Settings option

`option` = `ON||OFF` : Means that you can't have this feature at the same time?

If it is a `ON||OFF` (`input[type=checkbox]`)  option, it is completely unnecessary, which means that it is difficult for users to choose.... Cannot be temporarily worked around, you cannot go to the option page every time you need to turn ON/OFF, and then ON/OFF again Right...


👇 Like this, users can have 2 functions at the same time! 
![212_21080406](https://user-images.githubusercontent.com/10995302/108611638-60afb900-741b-11eb-8c21-604db2a6aa38.png)

If necessary, you can add the hotkey for this function again at `chrome://extensions/shortcuts#:~:text=Save for later`

**Comment by dnknn** [comment]
> then we would also need to have to open a new tab to keep the browser running.

open `chrome://newtab/` by default, if can , also allow to custom... (Of course, this is not the most important)

Like https://chrome.google.com/webstore/detail/faminaibgiklngmfpfbhmokfmnglamcm
![image](https://user-images.githubusercontent.com/10995302/108617125-c36e7800-744e-11eb-94f3-16ad6074c3bc.png)

**Comment by allen505** [comment]
Yes, so they have to choose between either via the options. I think that will be what most people are looking for. Depending on the feedback from there, let's see if further action is necessary. Adding a new button or a left click, right click mechanics will complicate things according to me.  
Yes, opening a new tab on closing the entire window is the plan for now.

**Comment by dnknn** [comment]
> Yes, so they have to choose between either via the options. I think that will be what most people are looking for.

No, if that’s the case, I don’t think it makes much sense, because people can't have it at the same time, it means that there is no way to maximize functional efficiency!

Just think about it, sometimes you need Save, and sometimes Save&&Close, because people don't always want a specific one, but based on the temporary needs of the scene.

If it is the ON/OFF option, it will not be able to meet this demand.



> Adding a new button or a left click, right click mechanics will complicate things according to me.

`left click, right click` :  You can add a `tooltip` to the `Save` button `[title="R-click: save && close"]`

When the user the mouse hover the button, tooltip info will be displayed, then the user will know the function.
